abel/basex.py

```diff
@@ -134,7 +134,7 @@ def basex_transform(data, nbf='auto', reg=0.0, basis_dir='./', dr=1.0, verbose=T
     recon = basex_core_transform(data, Ai, dr)
 
     if data_ndim == 1:  # taking the middle row, since the rest are zeroes
-        recon = recon[recon.shape[0] - recon.shape[0]//2 - 1] #??
+        recon = recon[recon.shape[0] - recon.shape[0]//2 - 1]  # ??
```

Comment:
What is this supposed to do?
`data_ndim == 1` means that recon.shape[0] would be 1, and there are no other rows. Should this be just `recon = recon[0]`?